<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>0.000000000000000</double>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QLabel" name="label_dro_z">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_dtg_x">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>DTG X</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_dro_uh_z">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>UH</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_dro_y">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_vel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>Vel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_dro_x">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5" rowspan="2">
      <widget class="QToolButton" name="toolButton_fit_all">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/vendor/icons/zoom_extend.svg</normaloff>:/vendor/icons/zoom_extend.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_dtg_y">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>DTG Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_dro_dtg_z">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_dro_uh_y">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>UH</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_dro_uh_x">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>UH</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_dro_dtg_y">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_dro_vel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_y">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_dro_dtg_x">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_dtg_z">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>DTG Z</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_x">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_z">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0,0);</string>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
